dimagi/pages/views/core.py

```diff
@@ -16,6 +16,13 @@
 from dimagi.utils.wordpress_api import get_json
 from dimagi.pages.models.partners import latestPartners
 
+from dimagi.data.blog import (
+    get_category_by_slug
+)
+from dimagi.pages.views.blog import (
+    _get_posts,
```

Comment:
these are private methods and should not be used outside of `dimagi.pages.views.blog`!

Reply:
Got it! I'll copy these methods.

Reply:
@nemanja1040 no...copying methods is not the right answer either! duplicating code should never be the right answer...